lib/eex/lib/eex/engine.ex

```diff
@@ -133,11 +144,23 @@ defmodule EEx.Engine do
     end
   end
 
+  def handle_expr(_buffer, "/", _expr) do
+    raise_marker_not_implemented("/")
+  end
+
+  def handle_expr(_buffer, "|", _expr) do
+    raise_marker_not_implemented("|")
+  end
+
   def handle_expr(buffer, "", expr) do
     quote do
       tmp2 = unquote(buffer)
       unquote(expr)
       tmp2
     end
   end
+
+  defp raise_marker_not_implemented(marker) do
+    raise EEx.SyntaxError, message: "handle_expr/3 not implemented for <%#{marker} %>"
```

Comment:
The line is long, so for me long syntax looks better. _De gustibus non est disputandum_ :)